client/components/plan-storage/README.md

```diff
@@ -0,0 +1,64 @@
+Plan Storage
```

Comment:
Nice component! I think we should put it in `my-sites` instead, though, as `client/components` should eventually only hold devdocs UI components. (Similar example, `site` component in `my-sites/site`.)